packaging/os/pacman.py

```diff
@@ -130,6 +151,27 @@ def update_package_db(module):
     else:
         module.fail_json(msg="could not update package db")
 
+def upgrade(module):
+    cmdupgrade = "pacman -Suq --noconfirm"
```

Comment:
Well, before upgrading you should always sync with "y", so the command should be "pacman -Syuq --noconfirm". Doing a update_cache=yes is not necessary because an "-Syy" takes longer than needed.